src/containers/form/AppSagas.js

```diff
@@ -0,0 +1,140 @@
+/*
+ * @flow
+ */
+
+import { put, take, takeEvery } from 'redux-saga/effects';
+
+import { APP_NAMES } from '../../shared/Consts';
+
+import {
+  fetchApp,
+  fetchAppConfigs,
+  fetchAppTypes,
+  fetchEntityDataModelProjection
+} from '../../core/lattice/LatticeActionFactory';
+
+import {
+  LOAD_APP,
+  LOAD_CONFIGURATIONS,
+  loadApp,
+  loadConfigurations
+} from './AppActionFactory';
+
+import type { SequenceAction } from '../../core/redux/RequestSequence';
+
+/*
+ * helper functions
+ */
+
+function matchFetchProjectionResponse(fetchProjectionAction :SequenceAction) {
+  return (anAction :Object) => {
+    return (anAction.type === fetchEntityDataModelProjection.SUCCESS && anAction.id === fetchProjectionAction.id)
+      || (anAction.type === fetchEntityDataModelProjection.FAILURE && anAction.id === fetchProjectionAction.id);
+  };
+}
+
+function matchFetchAppResponse(fetchAppAction :SequenceAction) {
+  return (anAction :Object) => {
+    return (anAction.type === fetchApp.SUCCESS && anAction.id === fetchAppAction.id)
+      || (anAction.type === fetchApp.FAILURE && anAction.id === fetchAppAction.id);
+  };
+}
+
+function matchFetchAppTypesResponse(fetchAppTypesAction :SequenceAction) {
+  return (anAction :Object) => {
+    return (anAction.type === fetchAppTypes.SUCCESS && anAction.id === fetchAppTypesAction.id)
+      || (anAction.type === fetchAppTypes.FAILURE && anAction.id === fetchAppTypesAction.id);
+  };
+}
+
+function matchFetchAppConfigsResponse(fetchAppConfigsAction :SequenceAction) {
+  return (anAction :Object) => {
+    return (anAction.type === fetchAppConfigs.SUCCESS && anAction.id === fetchAppConfigsAction.id)
+      || (anAction.type === fetchAppConfigs.FAILURE && anAction.id === fetchAppConfigsAction.id);
+  };
+}
+
+/*
+ * sagas
+ */
+
+ export function* loadAppWorker() :Generator<*, *, *> {
+
+   try {
+
+     yield put(loadApp.request());
```

Comment:
`redux-reqseq@0.1.0` introduces a breaking change that requires these helper functions to take in an `id` as the first parameter.

```
// watcher sagas pass in the redux action to the worker sagas
function* loadAppWorker(action) {
  try {
    yield put(loadApp.request(action.id));
    ...
    yield put(loadApp.success(action.id, response));
  }
  catch (error) {
    yield put(loadApp.failure(action.id, error));
  }
  finally {
    yield put(loadApp.finally(action.id));
  }
}
```